site/layouts/_default/single.html

```diff
@@ -1,14 +1,19 @@
 {{ define "main" }}
   <header class="py-5 border-bottom">
-    <div class="container pt-md-1 pb-md-4">
+  <div class="container-xxl bd-gutter pt-md-1 pb-md-4">
       <div class="row">
         <div class="col-xl-8">
           <h1 class="bd-title mt-0">{{ .Title | markdownify }}</h1>
           <p class="bd-lead">{{ .Page.Params.Description | markdownify }}</p>
           {{ if eq .Title "Examples" }}
-          <div class="d-flex flex-column flex-sm-row">
-            <a href="{{ .Site.Params.download.dist_examples }}" class="btn btn-lg btn-bd-primary" onclick="ga('send', 'event', 'Examples', 'Hero', 'Download Examples');">Download examples</a>
-            <a href="{{ .Site.Params.download.source }}" class="btn btn-lg btn-outline-secondary mt-3 mt-sm-0 ms-sm-3" onclick="ga('send', 'event', 'Examples', 'Hero', 'Download');">Download source code</a>
+          <div class="d-flex flex-column flex-md-row gap-3">
+            <a href="{{ .Site.Params.download.dist_examples }}" class="btn btn-lg bd-btn-lg btn-bd-primary d-flex align-items-center justify-content-center fw-semibold" onclick="ga('send', 'event', 'Examples', 'Hero', 'Download Examples');">
+              <svg class="bi me-2"><use xlink:href="#box-seam"></use></svg>
```

Comment:
Missing `aria-hidden="true"` here.
```suggestion
              <svg class="bi me-2" aria-hidden="true"><use xlink:href="#box-seam"></use></svg>
```